pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1853,10 +1853,11 @@ init_context(void *ctx, event_context_t *context, struct task_struct *task, u32
     return 0;
 }
 
-static __always_inline task_info_t *init_task_info(u32 tid, u32 pid)
+static __always_inline task_info_t *init_task_info(u32 tid, u32 pid, bool kthread)
 {
-    int zero = 0;
-    scratch_t *scratch = bpf_map_lookup_elem(&scratch_map, &zero);
+    int scratch_map_idx = kthread ? 1 : 0; // second buffer for kthread ctx
```

Comment:
Not necessarily a kthread context I believe

Reply:
https://github.com/aquasecurity/tracee/pull/2616#issuecomment-1400636143